### DISCUSSION TITLE
Actions secrets : "Failed to add secret. Please try again."

### DISCUSSION THREAD

**Discussion by h-ql** [community/community#42733] [Actions] [answered]
i have trouble can't create Actions Secrets 
i tried many time with correct naming rules , other browser ... can't able to figure out why
anybody have any idea why or what i missed?

![png1](https://user-images.githubusercontent.com/74464965/209736200-1c42e9cb-5633-41a0-938d-bf1fc9f34962.png)
![png2](https://user-images.githubusercontent.com/74464965/209736204-63742448-414b-41f5-9c94-1bad23e66cd9.png)

**Comment by h-ql** [comment, marked as answer]
Answered

**Comment by ebran-flow** [comment]
Had the same issue, I just wanted to create all the fields and enter the values later. GitHub however requires the values to be not empty.
